### PR TITLE
Configuration files

### LINKED ISSUE #54: Config file
Using [configparser](https://docs.python.org/3/library/configparser.html) python library create a new file called config.ini where put all the configurations used on yapi.

### PR DESCRIPTION
**Pull request by Wabri**
Closes #54

### CODE REVIEW

File: config.ini [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+[REMOTE]
+raw_location = https://raw.githubusercontent.com/YetAnotherPackageInstaller/
+origin_location = https://github.com/YetAnotherPackageInstaller/YAPI
+branch = master
+zip = ${REMOTE:origin_location}/archive/${REMOTE:branch}.zip
+
+[PACKAGES]
+ignore = test.sh, updateYapiScripts.sh
+installed =
```
Comment (IanDuncanT) [inline config.ini]: What is the value of installed supposed to be?

File: platform.ini [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[COMMON]
+main_url = https://github.com/YetAnotherPackageInstaller
+
+[DARWIN]
```
Comment (IanDuncanT) [inline platform.ini]: Is there a reason for a MacOS and Debian package location in YAPI Origin? I thought this was a single platform for debian. Please explain this config.

File: config.ini (hunk 1 above)
Comment (Wabri) [reply config.ini]: I want to track all the packages installed with yapi, but maybe it is better to do with a cache file like what i've done with scripts.bin

File: platform.ini (hunk 2 above)
Comment (Wabri) [reply platform.ini]: Maybe in a future developing this can be useful, I didn't think about yapi as multiplatform but who knows

**Pull request merged**